src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -377,53 +359,118 @@ def work_for_vts_rsc(vts, ctx):
           ' (',
           ctx.target.address.spec,
           ').')
+        # things to do
+        # collect jar dependencies outlines
+        # collect non-java dependencies
+        # metacp remaining java dependencies using jar and non-java deps as deps
+        # rsc sources,
+        #   providing metacp'd
+        #    - jar deps
+        #    - non-java deps
+        #    - metacp'd java deps
+
+
+
+        strict_dependencies_for_target = list(DependencyContext.global_instance().dependencies_respecting_strict_deps(target))
```

Comment:
From what I can see, `strict_dependencies_for_target` don't have to be strict dependencies, do they? If my understanding is correct, perhaps this variable could be renamed?

Reply:
They are not necessarily strict_deps. So I could use a better name here. I'll update it.